waterlog/src/actions/WastageDaily.js

```diff
@@ -0,0 +1,35 @@
+import { FETCH_WASTAGE_DAILY, FETCH_WASTAGE_MONTHLY , FETCH_WASTAGE_SEASONALLY} from "./Types";
+import { Globals } from './../Globals';
+
+export const fetchWastageDaily =() => (dispatch) => {
+    fetch(`${Globals.API_URL}/api/segmentevents/dailywastage`) //Change to use either localhost/server
+      .then(res => res.json())
+      .then(dailywaste => {
+        dispatch({
+            type: FETCH_WASTAGE_DAILY,
+            payload: dailywaste
```

Comment:
Maybe make this camelCase

Reply:
dailyWaste, monthlyWaste and seasonWaste are now camel case

Reply:
Cool.